src/components/CStreamStatus/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+export enum StreamStatus {
+  ONGOING = 'ongoing',
+  PENDING = 'pending',
+  EXPIRED = 'expired',
+}
+
+const streamStatuses = [
+  { label: 'Ongoing', value: StreamStatus.ONGOING },
+  { label: 'Pending', value: StreamStatus.PENDING },
+  { label: 'Expired', value: StreamStatus.EXPIRED },
+];
+
+type CStreamStatusProps = {
+  onChange: (value: StreamStatus) => void;
+};
+
+const CStreamStatus = ({ onChange }: CStreamStatusProps) => {
+  const [activeStatus, setActiveStatus] = useState('ongoing');
```

Comment:
The defaultValue should be used from an enum and give the state a tpye declaration of StreamStatus type